public/app/core/components/TraceToLogs/TraceToLogsSettings.tsx

```diff
@@ -3,15 +3,19 @@ import {
   DataSourceJsonData,
   DataSourcePluginOptionsEditorProps,
   GrafanaTheme,
+  KeyValue,
   updateDatasourcePluginJsonDataOption,
 } from '@grafana/data';
 import { DataSourcePicker } from '@grafana/runtime';
 import { InlineField, InlineFieldRow, Input, TagsInput, useStyles, InlineSwitch } from '@grafana/ui';
 import React from 'react';
+import KeyValueInput from './KeyValueInput';
 
 export interface TraceToLogsOptions {
   datasourceUid?: string;
   tags?: string[];
+  mappedTags?: Array<KeyValue<string>>;
+  mapTagNamesEnabled?: boolean;
```

Comment:
What happens with tags that the user would like to be included but not mapped. Do they need explicit mappings to the same value? For example if I had tags `pod`, `application`, `service.name` and wanted to map only the last one. Would it require defining `pod=pod`, `application=application`, `service.name=service`?

Reply:
Right now, if no 'value' is defined, it uses the key as a fallback.